src/App.js

```diff
@@ -98,6 +98,15 @@ class App extends Component {
       const { showOrganisationSwitcher } = this.state;
       const breadcrumbs = this.computeBreadcrumb();
 
+      // Redirect for logging out.
+      // It takes the same base url as the current management client url.
+      let currentUrlBase = window.location.href.split("/management")[0];
+      // Redirect to staging for development
+      if (currentUrlBase.indexOf("localhost")) {
```

Comment:
.indexOf levert -1 op als de string niet voorkomt; als hij 0 oplevert is hij wel gevonden, aan het begin van de string. Dus hier had ` if (currentUrlBase.indexOf("localhost") > -1) {` o.i.d. moeten staan; maar zie onder.

Reply:
Goeie comments, dankjewel! :)